src/lib/rules/http-cache/http-cache.ts

```diff
@@ -344,7 +346,7 @@ const rule: IRuleBuilder = {
             });
 
             if (!matches) {
-                const message: string = `No patterns for file revving match ${resource}`;
+                const message: string = `No patterns for file revving match ${resource}. See docs to add custom pattern.`;
```

Comment:
I think the initial confusion was about what file revving. Maybe we should make that more clear use something such as: `No patterns for cache busting using filename/path based revving match...`?

Reply:
What about:

> No configured patterns for cache busting match ${resource}. See docs to add a custom one.

Reply:
:+1: